oaknational/Oak-Web-Application
# Added curriculum "computer science" cycle 2 conditions

## Description
> [!IMPORTANT]  
> This is branched off of https://github.com/oaknational/Oak-Web-Application/pull/2820 so that PR must land first

Added special cases for curriculum "computer science" cycle 2

<img width="300" alt="Screenshot 2024-09-25 at 16 06 33" src="https://github.com/user-attachments/assets/fd30b3d6-40fe-4ffe-9927-369f7484cb49">
<img width="300" alt="Screenshot 2024-09-25 at 16 06 18" src="https://github.com/user-attachments/assets/ba87b3b1-93c6-4091-a716-7f1b2da537ed">
<img width="300" alt="Screenshot 2024-09-25 at 16 07 04" src="https://github.com/user-attachments/assets/4873da3b-a98a-4b9e-907c-21f246d2d9d1">


## Issue(s)

Fixes

 - `CUR-839`
 - `CUR-838`
 - `CUR-837`



## How to test
Cycle 1

1. Go to https://deploy-preview-2822--oak-web-application.netlify.thenational.academy
2. Check the docx and visualiser for regressions

Cycle 2

 1. Start app locally with `ENABLE_CYCLE_2` enabled
 2. Go to computing in the visualiser and check for functionality mentioned in linked ticket
